### PR TITLE
Update docs to include info about PR check triggering

### PR DESCRIPTION
**Pull request by kmahone**
Cleans up some outdated info about the different Pipelines.
Also updates list of supported OSes with their informal name (e.g. "Redstone 1") for added clarity.

### CODE REVIEW

**Comment by kmahone** [comment]
Ah! The sweet smell of automated checks completing in seconds!

File: docs/contribution_workflow.md [hunk 1]
```diff
@@ -73,21 +73,19 @@ discussing it first: see the [New Feature or API Process](feature_proposal_proce
 
 Each pull request to `master` must pass the following checks.
 
-Pull requests into a fork will not trigger all of these checks, but you can manually request them 
-individually from the pipeline definition page.
+Pull requests from a fork will not automatically trigger all of these checks. A member of the WinUI 
+team can trigger the Azure Pipeline checks by commenting "/azp run" on the PR. The Azure Pipelines
```
Comment (chrisglein) [inline docs/contribution_workflow.md]:
```suggestion
team can trigger the Azure Pipeline checks by commenting `/azp run` on the PR. The Azure Pipelines
```

File: README.md [hunk 2]
```diff
@@ -65,12 +65,12 @@ The Microsoft.UI.Xaml 2.2 NuGet package requires your project to have TargetPlat
 
 Your app's users can be on any of the following supported Windows versions:
 * Windows Insider Previews
-* May 2019 Update (18362)
-* October 2018 Update (17763)
-* April 2018 Update (17134)
-* Fall Creators Update (16299)
-* Creators Update (15063)
-* Anniversary Update (14393)
+* May 2019 Update (18362 aka "19H1')
```
Comment (jesbis) [inline README.md]:
```suggestion
* May 2019 Update (18362 aka "19H1")
```

File: README.md [hunk 3]
```diff
@@ -65,12 +65,12 @@ The Microsoft.UI.Xaml 2.2 NuGet package requires your project to have TargetPlat
...
+* October 2018 Update (17763 aka "Redstone 5')
```
Comment (jesbis) [inline README.md]:
```suggestion
* October 2018 Update (17763 aka "Redstone 5")
```

File: README.md [hunk 4]
```diff
@@ -65,12 +65,12 @@ The Microsoft.UI.Xaml 2.2 NuGet package requires your project to have TargetPlat
...
+* April 2018 Update (17134 aka "Redstone 4')
```
Comment (jesbis) [inline README.md]:
```suggestion
* April 2018 Update (17134 aka "Redstone 4")
```

File: README.md [hunk 5]
```diff
@@ -65,12 +65,12 @@ The Microsoft.UI.Xaml 2.2 NuGet package requires your project to have TargetPlat
...
+* Fall Creators Update (16299 aka "Redstone 3')
```
Comment (jesbis) [inline README.md]:
```suggestion
* Fall Creators Update (16299 aka "Redstone 3")
```

File: README.md [hunk 6]
```diff
@@ -65,12 +65,12 @@ The Microsoft.UI.Xaml 2.2 NuGet package requires your project to have TargetPlat
...
+* Creators Update (15063 aka "Redstone 2')
```
Comment (jesbis) [inline README.md]:
```suggestion
* Creators Update (15063 aka "Redstone 2")
```

File: README.md [hunk 7]
```diff
@@ -65,12 +65,12 @@ The Microsoft.UI.Xaml 2.2 NuGet package requires your project to have TargetPlat
...
+* Anniversary Update (14393 aka "Redstone 1')
```
Comment (jesbis) [inline README.md]:
```suggestion
* Anniversary Update (14393 aka "Redstone 1")
```

**Pull request merged**